pkg/ipcache/metadata.go

```diff
@@ -312,8 +311,6 @@ func (ipc *IPCache) RemoveLabelsExcluded(
 	lbls labels.Labels,
 	toExclude map[string]struct{},
 	src source.Source,
-	updater ipcacheTypes.PolicyHandler,
-	triggerer ipcacheTypes.DatapathHandler,
```

Comment:
Minor nit: Looks like these removed parameters were ultimately not used already, so maybe this commit could be squashed with the one that actually removed the use of these parameters in `removeLabelsFromIPs()`?